### ISSUE TITLE
[Feat] - Template modification

### ISSUE THREAD

**Issue by jenaye** [CMEPW/221b#4]
to prevent the packer from being overflagged, flags should be added to modify the final result slightly
for example, the possibility of modifying the duration of sleep
or store shellcode in .text instead of .data, or even another section

**Comment by TomChv** [comment]
We can add a flag to provide your own go code template, would that work?

**Comment by jenaye** [comment]
No, I was thinking more of several templates on our side, some where the shellcode will be in .text, others where it will be in .data or .rsrc
Create a new section can also be a good thing

**Comment by TomChv** [comment]
I see, I have no clue how to do this in Go but we can work on it
